scripts/data/vestings.json

```diff
@@ -0,0 +1,67 @@
+{
+    "statics": {
+        "origin": "!!! address that pays bsx as they vest; treasury address I guess !!!"
+    },
+    "vestings": [
+        {
+            "destination": "J9ReHt4cjND2EUnkHf2XqgzHNWPmwHL6s6w3Zqa1UgmumfA",
+            "schedule": {
+                "amtToBeVestedOverTime": "20250027965461027",
+                "start": "!!! end block of BSX LBP !!!",
+                "period": "1",
+                "per_period": "20250027965461027",
+                "period_count": "1"
+            }
+        },
+        {
+            "destination": "J9ReHt4cjND2EUnkHf2XqgzHNWPmwHL6s6w3Zqa1UgmumfA",
+            "schedule": {
+                "amtToBeVestedOverTime": "47250065252742395",
+                "start": "9071999",
+                "period": "1",
+                "per_period": "9765636468",
+                "period_count": "4838401"
+            }
+        },
```

Comment:
you see every addresss has two of these 'vestings' objects,  one for the 30% an address gets to claim up-front,  one for the other 70% that an address gets to claim during the term of the lease